oysttyer-2.9.1

woffs opened the pull request:
###### Motivation for this change

provide oysttyer, a cmdline twitter client in perl

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

woffs committed:
oysttyer: init at 2.9.1

A Perl Console Twitter Client

cillianderoiste commented:
Thanks @woffs, could you add yourself as the maintainer and (assuming you've tested that it works) check the relevant boxes, cheers!

woffs commented:
@cillianderoiste I have added maintainer and license, read CONTRIBUTING.md and squashed the commits into one with a conforming message. Hope this is the right way to go.
